### DISCUSSION TITLE
FRC in Just

### DISCUSSION THREAD

**Discussion by jgarte** [casey/just#2534] [Q&A] [answered]
How would I do the following in just?

https://github.com/leahneukirchen/px/blob/master/Makefile#L24

The equivalent of FRC or force rebuild all deps?
FRC

**Comment by casey** [comment, marked as answer]
I think something like FRC isn't required, because all `just` recipes are `.PHONY` recipes, so they always run, regardless of which files are or aren't present and how fresh they are.

**Reply by jgarte** [reply]
Thanks for the info.
